### ISSUE TITLE
Server/channel not displayed for chats

### ISSUE THREAD

**Issue by alexolog** [EionRobb/purple-discord#140]
When somebody from a channel I'm on sends me a private message, Pidgin just shows a a long number as the window title, giving me absolutely no indication where did this person come from.

Even worse, the same thing happens when I get a notification because somebody tagged "@everybody" (or me) on a channel.  A windows pops up with a long numeric id the title, giving me no indication what server and channel the discussion takes place on.

Please note that I keep "auto-create rooms" turned off, because I am not interested in many of the channels on some of the servers I'm on (especially dynamically created ones).

In short:
- For private chats with people who not set up as buddies in Pidgin, display the Discord server.
- For public chats on channels that are not set up as rooms  in Pidgin, display the Discord server and channel.

**Comment by EionRobb** [comment]
Unfortunately, neither are possible without auto-adding chats to the buddy list, as libpurple/pidgin has no way to alias a group chat that aren't in buddy lists

**Comment by alexolog** [comment]
Doesn't the server send this information?

**Comment by EionRobb** [comment]
Yes and the plugin stores that info, but it can only give an ID to libpurple, and its up to the UI (like Pidgin) to use the chat on the buddy list to fetch an alias for that ID

**Comment by alexolog** [comment]
Hmmm...  I guess it is possible to enable the auto-add, keep that group collapsed and copy the "interesting" channels to other groups

**Comment by alexolog** [comment]
1) It will not help with private chat.
Use case: a person who is not a buddy sends me a private message.  Using the Discord web interface I can check "mutual servers" on that person's profile, but I can't see a way to do it in Pidgin.

2) The auto-add is painful.
- If I change the alias of an auto-added channel, it gets re-added with the original name (so now I have two of them).  So if I have two channels with the same name (on different servers), I can't differentiate between them.
- If I move a channel to a different group, it also gets re-added.

**Comment by bobbybee** [comment]
Try right click -> get info?

**Comment by alexolog** [comment]
Hmm, it works.  Two points for Griffindor!

**Comment by alexolog** [comment]
There is nothing to right-click on, but Ctrl-O works.
